src/app/digital/models/DigitalWire.ts

```diff
@@ -74,4 +74,4 @@ export class DigitalWire extends Wire {
     public getIsOn(): boolean {
         return this.isOn;
     }
-}
+}
```

Comment:
nit: put the blank last line back so that this file basically just becomes un-edited